### ISSUE TITLE
Alpha value in loss

### ISSUE THREAD

**Issue by sudhakar-sah** [jinfagang/keras_ssd#1]
Hi Could you train the model well. 
There seems to be a problem 
  ssd_loss = SSDLoss(neg_pos_ratio=3, n_neg_min=0, alpha=0.1)
why is alpha chosen as 0.1. I think in the paper both alpha and beta are supposed to be 1.0 (same weightage for classification and bounding boxes loss)

**Comment by lucasjinreal** [comment]
Hi, this is actually seems not work at all, leave this, maybe TensorFlow is better.

**Comment by sudhakar-sah** [comment]
Well, it works and it works pretty well. I think the alpha value is not correct. Also, it needs more tuning to make it work. I wanted to ask, do you think that there is convergence difference in Keras vs. Tensorflow ?

**Comment by lucasjinreal** [comment]
You keep tuning...
